packages/components/app/styles/components/pagination.scss

```diff
@@ -33,7 +54,13 @@
 
 .hds-pagination-nav {
   display: flex;
-  grid-area: center;
+  grid-area: nav;
+
+  @media screen and (max-width: 1000px) {
+    justify-content: center;
+    order: -1;
+    width: 100%;
```

Comment:
maybe add a comment here, to explain why this `width: 100%` is needed? (I know it, you know it, but may help the next person that works on this code)